util/masking.py

```diff
@@ -271,20 +270,24 @@ def generate(self, imageset):
 
                     if region.circle is not None:
                         xc, yc, radius = region.circle
-                        logger.info("Generating circle mask:")
-                        logger.info(" panel = %d" % region.panel)
-                        logger.info(" xc = %d" % xc)
-                        logger.info(" yc = %d" % yc)
-                        logger.info(" radius = %d" % radius)
+                        logger.debug(
+                            "Generating circle mask:\n"
+                            + f" panel = {region.panel}\n"
+                            + f" xc = {xc}\n"
+                            + f" yc = {yc}\n"
+                            + f" radius = {radius}"
```

Comment:
Assuming the split lines are deliberate, I think the `+` are redundant here e.g. 
```suggestion
                            f" panel = {region.panel}\n"
                            f" xc = {xc}\n"
                            f" yc = {yc}\n"
                            f" radius = {radius}"
```